pkg/ccl/backupccl/restore_job.go

```diff
@@ -660,6 +660,27 @@ func shouldPreRestore(table *tabledesc.Mutable) bool {
 	return ok
 }
 
+// eligibleInProgressImport returns true if the descriptor represents a table with an in progress
+// import that started in a cluster finalized to version 22.2.
+func eligibleInProgressImport(
+	ctx context.Context, p sql.JobExecContext, desc catalog.Descriptor,
+) (bool, error) {
+	table, ok := desc.(catalog.TableDescriptor)
+	if !ok {
+		return false, nil
+	}
+
+	if table.GetInProgressImportStartTime() == 0 {
+		return false, nil
+	}
+	if !p.ExecCfg().Settings.Version.IsActive(ctx, clusterversion.Start22_2) {
```

Comment:
As we discussed on our call, I think this version check is a little disconnected from what it is guarding against. If we don't want descriptors to have a GetInProgressImportStartTime set in a mixed version we should guard it where we set it (I think you already do), instead of where we read it. I would delete this but if you feel better with it here its okay too.

Reply:
Done. Deleted.